docker-compose.yml

```
@@ -47,3 +47,23 @@ services:
     command: bundle exec sidekiq -r ./worker.rb
     depends_on:
       - redis
+  deqm_test_server:
+    depends_on:
+      - mongo
+    image: tacoma/deqm-test-server:connectathon
```

```suggestion
    image: tacoma/deqm-test-server:latest
```
suggest using the `latest` tag that is auto-built on deqm-test-server docker hub now

---

ah smart- forgot about that one